classes/Tab.php

```diff
@@ -84,7 +81,6 @@ class TabCore extends ObjectModel
             'route_name' => ['type' => self::TYPE_STRING, 'required' => false, 'size' => 256],
             'active' => ['type' => self::TYPE_BOOL, 'validate' => 'isBool'],
             'enabled' => ['type' => self::TYPE_BOOL, 'validate' => 'isBool'],
-            'hide_host_mode' => ['type' => self::TYPE_BOOL, 'validate' => 'isBool'],
```

Comment:
Since you remove this field you need to update the `db_strcture.sql` file as well
And a PR on autoupgrade will be needed as well

Reply:
Or maybe not because it's a Doctrine Entity right? Then only autoupgrade PR is needed